masterlist.yaml

```
@@ -4403,7 +4403,7 @@ plugins:
       - crc: 0x06D951BD
         util: 'TES4Edit v4.0.4b'
   - name: 'Improved NPC Faces for OCOv2.esp'
-    after: [ 'OCO Cleaned - Vanilla Hair.esp' ]
+    after: [ '(OCO Cleaned - Vanilla Hair|OCO Vanilla Hair)\.esp' ]
```

Regexes aren't supported for the `after` key, so you have to explicitly enumerate each plugin (for future reference, LOOT's metadata syntax is [fully documented](https://loot-api.readthedocs.io/en/stable/metadata/data_structures/plugin.html)):

```suggestion
    after:
      - 'OCO Cleaned - Vanilla Hair.esp'
      - 'OCO Vanilla Hair.esp'
```